chemprop/features/features_generators.py

```diff
@@ -131,7 +131,9 @@ def rdkit_2d_normalized_features_generator(mol: Molecule) -> np.ndarray:
         """Mock implementation raising an ImportError if descriptastorus cannot be imported."""
         raise ImportError('Failed to import descriptastorus. Please install descriptastorus '
                           '(https://github.com/bp-kelley/descriptastorus) to use RDKit 2D normalized features.')
-
+except AttributeError as e:
+    raise e('`descriptastorus==2.6.1` is incompatible with `scipy<1.9`. Please try changing '
```

Comment:
sorry another nit: is it possible to check the descriptastorus and scipy versions in this error message? The message assumes a user has specific versions, but we should actually check that before prescribing a reason for the error. You should be able to check using something like `scipy.__version__`